Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -80,6 +80,7 @@
   parent: ClothingHandsGlovesBoxingBlue
   id: ClothingHandsGlovesBoxingRigged
   name: blue boxing gloves
+  suffix: Rigged
   description: Blue gloves for competitive boxing. Feels like they're heavier than usual...
```

```suggestion
  description: Regal blue gloves, with a nice gold trim. Swanky.
```

I agree with delta and these should be considered a stealth item.